tools/recipes/rt-app.recipe

```diff
@@ -9,11 +9,21 @@ build_jsonc() {
     make
 }
 
+build_numactl() {
+    cd numactl
+    ./autogen.sh
+    echo "./configure --host=$CONFIGURE_HOST --disable-shared --enable-static"
+    ./configure --host=$CONFIGURE_HOST --disable-shared --enable-static
+    make
+}
+
 build_rtapp() {
     cd rt-app
     export ac_cv_lib_json_c_json_object_from_file=yes
+    export ac_cv_lib_numa_numa_available=yes
     ./autogen.sh
-    ./configure --host=$CONFIGURE_HOST LDFLAGS="--static -L$BUILD_DIR/json-c/" CFLAGS="-I$BUILD_DIR" --with-deadline
+    ./configure --host=$CONFIGURE_HOST LDFLAGS="--static -L$BUILD_DIR/json-c/ -L$BUILD_DIR/numactl/" CFLAGS="-I$BUILD_DIR" --with-deadline
+    make clean
```

Comment:
Is `make clean` actually needed ? The expected behavior of `lisa-build-asset` is to make each build in a clean work tree (either a clean clone or a copy of a clean clone).

If `make clean` is required for the build to succeed, that either indicates that :
*  the build relies on something that stays shared outside of `lisa-build-asset` control
* that `lisa-build-asset` has a bug somehow
* that rtapp repo contains files that need to be removed before building (unlikely)

Reply:
I will remove make clean.